### ISSUE TITLE
[Feature] Add shader support for Tesseract

### ISSUE THREAD

**Issue by nicolasdeory** [SuperMartijn642/Tesseract#88]
**Is your feature request related to a problem? Please describe.**
The Tesseract block does not render the "end portal" effect on the inside faces when using any shader pack with Embeddium 0.2.19+mc.1.20.1 and forge 47.2.17. 
![image](https://github.com/SuperMartijn642/Tesseract/assets/1417708/2a01643a-bae6-41a7-a90a-1723e80b3126)

It looks like the approach you're following using MatrixStack and your CustomBlockEntityRenderer isn't playing well with shaders, but I'm not familiar enough to be able to delve further. Disabling shader makes the block render correctly.

**Describe the solution you'd like**
It would be cool to be able to have the block render correctly when using shaders.

**Comment by nicolasdeory** [comment]
For reference, end portal rendering seems to work fine, although they are two different mechanisms, albeit looking similar.

![image](https://github.com/SuperMartijn642/Tesseract/assets/1417708/b50943c1-66c5-4586-abb0-1e88337ff041)

**Comment by SuperMartijn642** [comment]
I already just copied the rendering for the tesseracts from the end portal.

> ... when using any shader pack with Embeddium 0.2.19+mc.1.20.1 and forge 47.2.17.

Embeddium doesn't do shaders, so I assume you are using Oculus. It seems Oculus (and Iris for that matter) just completely replaces the rendering of the end portal https://github.com/Asek3/Oculus/blob/1.20/src/main/java/net/coderbot/iris/mixin/MixinTheEndPortalRenderer.java.

**Comment by nicolasdeory** [comment]
Correct, my bad. Forgot to mention I'm obviously also running Oculus 1.6.13. I guess the only way would be to create your own mixin that interacts with Oculus in case it was present. What's interesting is that the issue only happens with shaders enabled. If disabled, the inside of the cube renders correctly.

**Comment by Appa1990** [comment]
Hello, any news or workarounds for this?
